models/intel/age-gender-recognition-retail-0013/age-gender-recognition-retail-0013.yml

```diff
@@ -0,0 +1,56 @@
+models:
```

Comment:
IMO, these files should have a fixed name, such as `accuracy-check.yml`. There are a couple reasons:

1. `age-gender-recognition-retail-0013.yml` doesn't communicate any new information, because it's already in a folder named `age-gender-recognition-retail-0013`. A name like `accuracy-check.yml` tells you that this is the configuration file for checking the accuracy of the model.
2. It would be consistent with `model.yml`.

Reply:
Done